ViewModel/Study/FDACache.cs

```diff
@@ -670,22 +708,27 @@ public List<ChildElement> GetChildrenOfParent(ParentElement element)
             {
                 retVal.AddRange(IASElementSets);
             }
-            if (element.GetType() == typeof(AltervativeOwnerElement))
+            else if (element.GetType() == typeof(AltervativeOwnerElement))
```

Comment:
Use is not type == type

Reply:
This is a duplicate case of the one below, they're both AltervativeOwnerElement

Reply:
Also after typing that classname, it has a spelling error, shouldn't it be AlternativeOwnerElement?

Reply:
Deleted duplicate and changed the name of the class.